### PR TITLE
arcaflow integration POC

### PR DESCRIPTION
**Pull request by tsebastiani**
This Draft PR is an integration proposal between Arcaflow and Kraken using the newly created python Pypi wheel.

### CODE REVIEW

File: scenarios/arcaflow/sysbench/input.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+kubeconfig: {}
+sysbench_cpumaxprime: 12000
```
Comment (chaitanyaenr) [inline scenarios/arcaflow/sysbench/input.yaml:2]: We need to expose two more options to the user: 1) Node to target on OpenShift/Kubernetes env 2) Percent CPU to hog given that we might not know exact number of cores available. Thoughts?

Comment (chaitanyaenr) [reply scenarios/arcaflow/sysbench/input.yaml:2]: Options exposed on the arca plugin side

File: scenarios/arcaflow/sysbench/input.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+kubeconfig: {}
```
Comment (chaitanyaenr) [inline scenarios/arcaflow/sysbench/input.yaml:1]: Can we read the kubeconfig from the location specified in the Kraken config vs having to define in this config? Thoughts?

File: scenarios/arcaflow/kill-pod/workflow.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            type_id: string
+        kill:
+          display:
+            description: the number of pods to kill
+            name: kill
+          type:
+            type_id: integer
+steps:
+  kill_pod:
+    plugin: quay.io/tsebastiani/arcaflow/kill-pod:latest
```
Comment (chaitanyaenr) [inline scenarios/arcaflow/kill-pod/workflow.yaml:34]: We might want to move this image to redhat-chaos quay org - https://quay.io/organization/redhat-chaos.

**Comment by chaitanyaenr** [comment]
Ran the sysbench workflow on an OpenShift cluster and it worked!! We can merge the integration in post testing the pod-kill plugin. TO DO in later pull requests: Replace litmus scenarios after tweaking the PR to add the node-selector and CPU percentage options to let users pick a node to hog the % CPU specified in the config.

File: scenarios/arcaflow/kill-pod/workflow.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
+input:
```
Comment (chaitanyaenr) [inline scenarios/arcaflow/kill-pod/workflow.yaml:1]: We might want to tweak the docs to walk through the workflow and how to tweak it as per user needs: https://github.com/redhat-chaos/krkn/blob/main/docs/pod_scenarios.md.

**Review by chaitanyaenr — APPROVED** [review]
LGTM. Just one comment: maybe we can add docs on the integration as to how users can trigger workflows or we can do that as part of the pod-kill workflow addition in the follow up PR? Thoughts?

**Comment by paigerube14** [comment]
Agree with Ravi we should add some documentation to the workflows that are in arcaflow and give more links to configuration details

**Pull request merged**